systemFlake.nix

```diff
@@ -5,14 +5,19 @@
 , supportedSystems ? flake-utils-plus.lib.defaultSystems
 , inputs
 
-, nixosConfigurations ? { }
 , sharedExtraArgs ? { }
-, nixosProfiles ? { }
 , channels ? { }
 , channelsConfig ? { }
 , sharedModules ? [ ]
 , sharedOverlays ? [ ]
 
+, nixosConfigurations ? { }
+  # nixosProfiles should be deprecated since profiles aren't nixos specific
+, nixosProfiles ? { }
+, configProfiles ? nixosProfiles
+, defaultConfigOutput ? "nixosConfigurations"
+, defaultConfigBuilder ? null
```

Comment:
This line is a bit treacherous, it implies that there is _no_ `defaultConfigBuilder`

Reply:
I think `null` and possibly a comment then dealing with the default later is much cleaner. Trying to make a default here is going to get unnecessary complex.

We could just add a comment, that `null` is not actually the default.

Reply:
dropped these lines now defaults are set in code. Prefer #19 for defaults as not to increase flake-utils-plus api surface.